.circleci/config.yml

```diff
@@ -7,15 +7,15 @@ jobs:
   build:
     docker:
       - image: circleci/openjdk:11.0.7-jdk-buster
-      
+
       - image: circleci/mysql:8.0.4
         environment:
           MYSQL_ROOT_PASSWORD : my-secret-pw
-          MYSQL_ROOT_HOST: %          
+          MYSQL_ROOT_HOST: localhost
```

Comment:
@chayim how is this change related to this PR?

Reply:
It's not - there are (several) issues with the mysql connection, and I accidentally added this in by merging my branch down. Removed + pushed back. Now it's just as broken as before - but without the supply issue. I'll open another PR to clean that.